pkg/util/taints/taints.go

```diff
@@ -0,0 +1,32 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package taints implements utilities for working with taints
+package taints
+
+import (
+	"k8s.io/api/core/v1"
+)
+
+// TaintExists checks if the given taint exists in list of taints. Returns true if exists false otherwise.
```

Comment:
Is this really a cloud-provider-gcp utility?

Reply:
It is used when creates a new cloud CIDR allocator [here](https://github.com/kubernetes/cloud-provider-gcp/blob/4b1a25888e3ca1aa8269a84a1c58cb9dcf2dbe38/pkg/controller/nodeipam/ipam/cloud_cidr_allocator.go#L123)